scripts/measures/2022/update-measures-util.ts

```diff
@@ -302,10 +302,13 @@ export function writeToFile(file: any, filePath: string) {
     fs.writeFileSync(path.join(appRoot + '', filePath), JSON.stringify(file, null, 2));
 }
 
-function updateBenchmarksMetaData(change: MeasuresChange): any {
+function updateBenchmarksMetaData(change: MeasuresChange, measuresJson: any): any {
+    const currentMeasure = _.find(measuresJson, { 'measureId': change.measureId });
+    const icdImpacted = change.icdImpacted ? change.icdImpacted : currentMeasure.icdImpacted;
+    const clinicalGuidelineChanged = change.clinicalGuidelineChanged ? change.clinicalGuidelineChanged : currentMeasure.clinicalGuidelineChanged;
     return {
-        isIcdImpacted: change.icdImpacted ? !!change.icdImpacted.length : false,
-        isClinicalGuidelineChanged: change.clinicalGuidelineChanged ? !!change.clinicalGuidelineChanged.length : false,
+        isIcdImpacted: icdImpacted ? !!icdImpacted.length : false,
+        isClinicalGuidelineChanged: clinicalGuidelineChanged ? !!clinicalGuidelineChanged.length : false,
```

Comment:
This change to the script fixes an issue where metadata booleans would be reset to false if either truncation or suppression was changed in the RC while the other was not.